en/orm/validation.rst

```diff
@@ -438,6 +438,11 @@ two rules::
     $rules->add($rules->validCount('tags', 3, '>=', 'You must have at least 3 tags'));
     $rules->add($rules->validCount('tags', 5, '<=', 'You must have at most 5 tags'));
 
+.. note::
+
+   When comparing via the ``<`` or ``<=`` operators, ``validCount()`` will return
+   ``false`` if you do not supply at least an empty list of - say - tags.
+
```

Comment:
Well, but the unexpected behavior only happens for < and <= as 0, '', [], null, !property_exists(), could all imho only be interpreted as 0.

Aka if you validate that there needs to be one or more profiles, then the rule fails like expected if the key is missing.

However `==` is also affected